roles/ceph-common/tasks/main.yml

```diff
@@ -36,10 +36,20 @@
   # Hard code this so we will skip the entire file instead of individual tasks (Default isn't Consistent)
   static: False
 
+- include: ./pre_requisites/prerequisite_redhat_install.yml
+  when:
+    - ansible_distribution == "RedHat"
+    - ansible_distribution_major_version >= 7
+    - not ceph_rhcs_iso_install
```

Comment:
Why not `ceph_rhcs` so we address both `ceph_rhcs_cdn_install` and `ceph_rhcs_iso_install`?